src/components/ScenarioParameters/components/ScenarioParametersInputs/components/TableDeleteRowsDialog/TableDeleteRowsDialog.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { DontAskAgainDialog } from '@cosmotech/ui';
+import PropTypes from 'prop-types';
+
+export const TableDeleteRowsDialog = (props) => {
+  const { t } = useTranslation();
+  const { open, onClose, onConfirm, SelectedRowsCount } = props;
```

Comment:
Can you rename `SelectedRowsCount` to `selectedRowsCount`? (without the capital S)